packages/chip/src/Chip/Chip.types.ts

```diff
@@ -0,0 +1,82 @@
+import { DarkModeProps, HTMLElementProps } from '@leafygreen-ui/lib';
+
+export const TruncationLocation = {
+  Start: 'start',
+  Middle: 'middle',
+  End: 'end',
+  None: 'none',
+} as const;
+export type TruncationLocation =
+  (typeof TruncationLocation)[keyof typeof TruncationLocation];
+
+export const Size = {
```

Comment:
After speaking to Rob, we actually think that Chips should have a `baseFontSize` prop rather than have a `size` prop, since that is the fork not actually the size. We want every Chip in Atlas, for example, to use the 13px Chip rather than the 16px.

Reply:
i see, ok, that should be easy to change.